README.md

```diff
@@ -5,6 +5,9 @@
 ExMachina makes it easy to create test data and associations. It works great
 with Ecto, but is configurable to work with any persistence library.
 
+> **This README is for the unreleased beta version of ExMachina**. Here are the 
```

Comment:
In this case it is for a beta version, but you are right that that probably isn’t as clear. I like your suggested wording. I’ll go with that (or something very close to it)

> On Apr 5, 2016, at 9:05 AM, Jason Draper <notifications@github.com> wrote:
> 
> In README.md <https://github.com/thoughtbot/ex_machina/pull/114#discussion_r58533746>:
> 
> > @@ -5,6 +5,9 @@
> >  ExMachina makes it easy to create test data and associations. It works great
> >  with Ecto, but is configurable to work with any persistence library.
> >  
> > +> **This README is for the unreleased beta version of ExMachina**. Here are the 
> How about something like:
> 
> This README follows master, which may not be the currently released version.
> 
> Not picky about the wording. My main concerns were to make it clear that sometimes the docs may be the same and also that this isn't necessarily a "beta" version since it's just master.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/ex_machina/pull/114/files/ff47d13c17c9c161ca42719fde8f682b611642d3#r58533746>